src/renderer/screens/KeyboardSelect.js

```diff
@@ -150,8 +150,8 @@ class KeyboardSelect extends React.Component {
         .find(...Hardware.serial)
         .then(async devices => {
           let supported_devices = [];
-          for (const device of devices) {
-            if (await focus.isDeviceSupported(device)) {
+          for (let device of devices) {
+            if (await focus.isDeviceSupported(device, device.device)) {
```

Comment:
Why is the `device.device` needed there? The focus part only takes one argument (`device`).

Reply:
@algernon 
Yeah, you are wright.
Sorry for delaying with response.

Thank you for asking!

There were on my test case I have provided and it works... with some changes in `API`.
So I sill have to test it properly, cause I found the bug opening the port - that was a reason to pushed (but harryed up with PR to resolve asap)

It looks like the `isDeviceSupported` as on ANSI as on ISO are opening the same time..

Give me please a time for a while to check it once again.
I will let you know.